plugins/modules/ag_replica.py

```diff
@@ -7,109 +7,109 @@
 DOCUMENTATION = r'''
 ---
 module: ag_replica
-short_description: Configures an availability group replica.
+short_description: Configures an availability group replica
 description:
-     - Configures an availability group replica.
+- Configures an availability group replica.
 version_added: 0.5.0
 options:
   sql_instance_replica:
     description:
-      - The SQL Server instance where of the replica to be configured.
+    - The SQL Server instance where of the replica to be configured.
     type: str
     required: true
   sql_username_replica:
     description:
-      - Username for SQL Authentication for the secondary replica.
+    - Username for SQL Authentication for the secondary replica.
     type: str
     required: false
   sql_password_replica:
     description:
-      - Password for SQL Authentication for the secondary replica.
+    - Password for SQL Authentication for the secondary replica.
     type: str
     required: false
   ag_name:
     description:
-      - Name of the Availability Group that will have the new replica joined to it.
+    - Name of the Availability Group that will have the new replica joined to it.
     type: str
     required: true
   endpoint:
     description:
-      - By default, this command will attempt to find a DatabaseMirror endpoint. If one does not exist, it will create it.
+    - By default, this command will attempt to find a DatabaseMirror endpoint. If one does not exist, it will create it.
     type: str
     required: false
     default: 'hadr_endpoint'
   endpoint_url:
     description:
-      - By default, the property Fqdn of Get-DbaEndpoint is used as EndpointUrl.
-        Use EndpointUrl if a different URL is required due to special network configurations.
-        EndpointUrl has to be an array of strings in format 'TCP://system-address:port', one entry for every instance.
+    - By default, the property Fqdn of Get-DbaEndpoint is used as EndpointUrl.
+      Use EndpointUrl if a different URL is required due to special network configurations.
+      EndpointUrl has to be an array of strings in format 'TCP://system-address:port', one entry for every instance.
```

Comment:
please look out for more CamelCase options to convert to I(snake_case)

```suggestion
    - By default, the property C(Fqdn) of C(Get-DbaEndpoint) is used as I(endpoint_url).
      Use I(endpoint_url) if a different URL is required due to special network configurations.
      I(endpoint_url) has to be an array of strings in format C(TCP://system-address:port), one entry for every instance.
```

Also 
> I(endpoint_url) has to be an array of strings
The parameter is marked as a single string, rather than `type: list` with `elements: str`, should this be changed? If so, does the code for comparing an existing object need to be updated?